detect_shell_rad: 0.5
agent_mass: 1.0
agent_radius: 0.2
velocity_max: 0.5
approach_dist: 0.25
k_a_ee:     [2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0]
k_c_ee:     [4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0]
k_r_ee:     [2.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0] 
k_r_force:  [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
k_d_ee:     [4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0]
k_manip:    [4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0]